packages/api-format/src/codec/sample.json

```diff
@@ -0,0 +1,2199 @@
+                  "type": "Box<T::Proposal>"
+                },
+                {
+                  "name": "value",
+                  "type": "T::Balance"
+                }
+              ],
+              "documentation": []
+            },
+            {
+              "id": "0x0001",
+              "name": "second",
+              "arguments": [
+                {
+                  "name": "origin",
+                  "type": "T::Origin"
+                },
+                {
+                  "name": "proposal",
+                  "type": "PropIndex"
+                }
+              ],
+              "documentation": []
+            },
+            {
+              "id": "0x0002",
+              "name": "vote",
+              "arguments": [
+                {
+                  "name": "origin",
+                  "type": "T::Origin"
+                },
+                {
+                  "name": "ref_index",
+                  "type": "ReferendumIndex"
+                },
+                {
+                  "name": "approve_proposal",
+                  "type": "bool"
+                }
+              ],
+              "documentation": []
+            },
+            {
+              "id": "0x0003",
+              "name": "start_referendum",
+              "arguments": [
+                {
+                  "name": "proposal",
+                  "type": "Box<T::Proposal>"
+                },
+                {
+                  "name": "vote_threshold",
+                  "type": "VoteThreshold"
+                }
+              ],
+              "documentation": []
+            },
+            {
+              "id": "0x0004",
+              "name": "cancel_referendum",
+              "arguments": [
+                {
+                  "name": "ref_index",
+                  "type": "ReferendumIndex"
+                }
+              ],
+              "documentation": []
+            }
+          ]
+        }
+      },
+      "storage": {
+        "prefix": "Democracy",
+        "functions": [
+          {
+            "name": "PublicPropCount",
+            "modifier": 1,
+            "type": "PropIndex",
+            "documentation": [
+              " The number of (public) proposals that have been made so far."
+            ]
+          },
+          {
+            "name": "PublicProps",
+            "modifier": 1,
+            "type": "Vec<(PropIndex, T::Proposal, T::AccountId)>",
+            "documentation": [
+              " The public proposals. Unsorted."
+            ]
+          },
+          {
+            "name": "DepositOf",
+            "modifier": 0,
+            "type": {
+              "key": "PropIndex",
+              "value": "(T::Balance, Vec<T::AccountId>)"
+            },
+            "documentation": [
+              " Those who have locked a deposit."
+            ]
+          },
+          {
+            "name": "LaunchPeriod",
+            "modifier": 2,
+            "type": "T::BlockNumber",
```

Comment:
These `T::Type` stuff is driving me a bit batty. The Rust basically bleeds through. I'm in favour of stripping any `T::` stuff and just not having it at all. (Especially with events not having them and even having things duplicated as `T::`. And then even in the modules, there are some without `T`, and really `T::AccountId` === `AccountId` for the API purposes)